### ISSUE TITLE
[CLOSED] 🚀 [Feature] Concept to introduce Gradido-ID

### ISSUE THREAD

**Issue by ulfgebhardt** [Zahlungsmittel/Zahlungsmittel#1633]
<a href="https://github.com/clauspeterhuebner"><img src="https://avatars.githubusercontent.com/u/86960882?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [clauspeterhuebner](https://github.com/clauspeterhuebner)**
_Monday Mar 07, 2022 at 23:55 GMT_
_Originally opened as https://github.com/gradido/gradido/issues/1574_

----

describe the introduction of the Gradido-ID and all its concerning aspects:

* DB-Migration
   - in Nutzertabelle Feld UUID einführen
   - in Nutzertabelle Feld Alias einführen
   - Versionsfeld für Passwort-Verschlüsselung
       * V1: Email
       * V2: UUID
   - Community-Tabelle mit Community-ID
	 
* Prozesse:
      - Registrierung / Login
      - Passwort-Verschlüsselung
	    * muss versioniert werden
		* mit Login wird dann ein Update der Verschlüsselung auf UUID umgestellt
		* 
      - Umzugsservice
	    * Prüfung auf existente UUID
		* Trigger für Weiterleitung
      - Identity-Mapper
	    * von email -> UUID
	    * von alias -> UUID
	    * von... -> UUID
      - Umbenennung der Community
      - Community/Nutzer (aus-)sperren
         * Blockchain als Doppelte unabhängige Datenbasis
	 * Node-Server Betrieb auf Community-Ebene + als Gemeinwohl einzelner Nerds
     -

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Tuesday Nov 29, 2022 at 08:30 GMT_

----

@clauspeterhuebner Can you please investigate this issue and make a followup ticket for stuff still missing?

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/clauspeterhuebner"><img src="https://avatars.githubusercontent.com/u/86960882?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [clauspeterhuebner](https://github.com/clauspeterhuebner)**
_Wednesday Nov 30, 2022 at 15:55 GMT_

----

@ulfgebhardt yes ok, i create one or more tickets with the still open topics...
Should i assign them to you or is it enough to shift them in the icebox lane?

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Wednesday Nov 30, 2022 at 15:56 GMT_

----

@clauspeterhuebner You can just put them into ice box or leave them in New Issues - I will take care in given time.

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/clauspeterhuebner"><img src="https://avatars.githubusercontent.com/u/86960882?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [clauspeterhuebner](https://github.com/clauspeterhuebner)**
_Wednesday Nov 30, 2022 at 16:07 GMT_

----

@ulfgebhardt nice...
i attach the new tickets with the Epic Gradido-ID, right? O ris there an other place, where they should documented?

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/clauspeterhuebner"><img src="https://avatars.githubusercontent.com/u/86960882?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [clauspeterhuebner](https://github.com/clauspeterhuebner)**
_Wednesday Nov 30, 2022 at 16:23 GMT_

----

@ulfgebhardt i assume, that the password encryption change from Joseji contains the requirements of https://github.com/gradido/gradido/blob/master/docu/Concepts/TechnicalRequirements/UC_Introduction_of_Gradido-ID.md#2nd-stage, right?

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Thursday Dec 01, 2022 at 21:20 GMT_

----

> 

I think stage2 is completed once the cleanup PR of @jjimenezgarcia is merged. The functionality is already done.
